tests/util/kubeUtils.go

```diff
@@ -225,15 +225,20 @@ func GetPodsName(n string) (pods []string) {
 
 // GetPodStatus gets status of a pod from a namespace
 func GetPodStatus(n, pod string) string {
-	status, err := Shell("kubectl -n %s get pods %s -o jsonpath='{.status.phase}'", n, pod)
+	status, err := Shell("kubectl -n %s get pods %s --no-headers", n, pod)
```

Comment:
doesn't seem to match the pr description ? in which case does .status.phase is different from the status column?